profiles/common/modules/custom/tmgmt_poetry/src/tmgmt_poetry.behat.inc

```diff
@@ -148,12 +148,14 @@ class TmgmtPoetrySubContext extends RawDrupalContext implements DrupalSubContext
 
 
   /**
-   * Get the Current Node Id in the URL.
+   * Get the Current Node Id.
    */
   private function getCurrentNodeId() {
-    $url = $this->getSession()->getCurrentUrl();
-    preg_match_all('/node\/(\d*)/', $url, $matches);
-    return (isset($matches[1][0]) ? $matches[1][0] : FALSE);
```

Comment:
Ok.

For your question, I say yes because in scenario using this method, they always implies one content saved just before and the last one is current one. Parallel processes implies different DB if my information is correct.

Now, while I am writing these lines, I concede that it is not CI agnostic . Then I will revert this commit. We will see for another solution in the future.